examples/pylab_examples/annotation_demo.py

```diff
@@ -97,7 +97,7 @@
     ax = fig.add_subplot(111, projection='polar')
     r = np.arange(0, 1, 0.001)
     theta = 2*2*np.pi*r
-    line, = ax.plot(theta, r, color='#ee8d18', lw=3)
+    line, = ax.plot(theta, r)
```

Comment:
I think we should probably take it off in both cases, actually, so it will take the new defaults.  (This example isn't about how to change the linewidth).